app/templates/components/smart-overflow.hbs

```diff
@@ -0,0 +1,3 @@
+<div class="description">
```

Comment:
@niranjan94 Since here we need to compare the height of the div with the span. I am not getting how will I access the div given by component?